.gitmodules

```diff
@@ -2,6 +2,10 @@
 	path = katacoda-scenarios
 	url = https://github.com/devonfw/katacoda-scenarios.git
```

Comment:
Please change this to https://github.com/devonfw-tutorials/katacoda-scenarios.git
This is the old URL of the repo and is only working because of github redirection magic